5_databases/week3/New_Query_1708896606910.sql

```diff
@@ -0,0 +1,198 @@
+-- Active: 1707334336522@@localhost@3306@meal_sharing
+-- Create the Reservation table
+CREATE TABLE Reservation (
+    id INT PRIMARY KEY, number_of_guests INT, meal_id INT, created_date DATE, contact_phonenumber VARCHAR(255), contact_name VARCHAR(255), contact_email VARCHAR(255), FOREIGN KEY (meal_id) REFERENCES Meal (id)
+);
+
+-- Create the Meal table
+CREATE TABLE Meal (
+    id INT PRIMARY KEY, title VARCHAR(255), description TEXT, location VARCHAR(255), meal_datetime DATETIME, max_reservations INT, price DECIMAL(10, 2), created_date DATE
+);
+
+-- Create the Review table
+CREATE TABLE Review (
+    id INT PRIMARY KEY, title VARCHAR(255), description TEXT, meal_id INT, stars INT, created_date DATE, FOREIGN KEY (meal_id) REFERENCES Meal (id)
+);
+
+-- Insert sample data into the Meal table
+INSERT INTO
+    Meal (
+        id, title, description, location, meal_time, max_reservations, price, created_date
+    )
+VALUES (
+        1, 'Italian Feast', 'A delicious Italian meal with pasta, pizza, and tiramisu.', '123 Main St', '2024-02-25 18:00:00', 10, 25.99, '2024-02-24'
```

Comment:
This is the point I was trying to make in my previous comment, if you had AUTO_INCREMENT in the meal table, you would not have to add the ID here , it will be generated automatically.